src/components/FormDatePicker.vue

```diff
@@ -82,12 +82,20 @@ export default {
       date: null,
     }
   },
+  created() {
+    let that = this;
+    Validator.register('after_min_date', function(value, requirement, attribute) {
```

Comment:
We should use an [arrow function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) here instead of having to use `that = this`.